frontend/src/metabase/dashboard/containers/AutomaticDashboardApp.jsx

```diff
@@ -144,10 +145,10 @@ class AutomaticDashboardApp extends React.Component {
             {parameters && parameters.length > 0 && (
               <div className="px1 pt1">
                 <Parameters
-                  parameters={parameters.map(p => ({
-                    ...p,
-                    value: parameterValues && parameterValues[p.id],
-                  }))}
+                  parameters={getValuePopulatedParameters(
```

Comment:
`getValuePopulatedParameters` doesn't set a `value` property if `parameter.id in parameterValues` is false, so the output here changes slightly; parameters without values won't have a `value: undefined` property. I don't think that is an issue, however, because we use `getValuePopulatedParameters` with the `Parameters` component elsewhere in the codebase.